fala/urls.py

```diff
@@ -2,7 +2,7 @@
 from django.conf import settings
 from django.conf.urls.static import static
 from django.urls import re_path
-from adviser.views import AdviserView, AccessibilityView
+from fala.apps.adviser.views import AdviserView, AccessibilityView
```

Comment:
I am surprised this is necessary. `adviser` is included in the [project's `INSTALLED_APPS`](https://github.com/ministryofjustice/fala/blob/master/fala/settings/base.py#L188-L190) and therefore should be available to import from by name. May be worth reverting this line and seeing if anything actually breaks, if it was added while trying other stuff out and if you haven't already.

Reply:
@exonian you are indeed correct that by removing this, that nothing will break. 

I made the change as my IDE (VS Code) was complaining to me, that there were missing imports. 
<img width="591" alt="Screenshot 2024-04-17 at 16 59 55" src="https://github.com/ministryofjustice/fala/assets/76905544/16ff022b-e7cf-4be2-abcc-066676bbeb06">

Having the long path, stop the complaints.

Reply:
Yeah I realised when I saw the same on Stephen's PR that it was likely IDEs. I wonder if there's a Django plugin that can help it understand that installed apps are on the Python path 🤔

Reply:
Potentially - I'll remove/revert this change in the mean time.